tap_tableau/tap.py

```diff
@@ -37,6 +37,7 @@
     EmbeddedDatasourcesMetadataStream,
     UsersMetadataStream,
     WorkbooksMetadataStream,
+    ColumnLineageMetadataStream,
```

Comment:
you'll need to add this to the import statement above too plz

Reply:
All done